src/main/kotlin/io/github/kjm015/kylenewer/commands/DieCommand.kt

```diff
@@ -0,0 +1,63 @@
+package io.github.kjm015.kylenewer.commands
+
+import com.jagrosh.jdautilities.command.Command
+import com.jagrosh.jdautilities.command.CommandEvent
+import java.util.*
+
+/**
+ * This command will let Kyle probably not die
+ * Again, it's mostly utter garbage. It's the thought that counts.
+ *
+ * @author tdickens1234
+ * @since 7/26/2018
+ */
+
+class DieCommand : Command() {
+    // Required constructor for every command
+    init {
+        this.name = "die"
+        this.aliases = arrayOf("kill.")
+        this.help = "Kyle will probably not die."
+    }
+
+    /**
+     * This command overrides the abstract method from the Command class.
+     * When this command is called, this is the stuff that actually happens.
+     *
+     *
+     * For this class, Kyle executes the die command, which for now, does not much
+     * [io.github.kjm015.kylenewer.util.MessageGenerator]
+     *
+     * @param event - The instance of the command that got called
+     */
+    override fun execute(event: CommandEvent) {
+        val reply = String.format("%s: %s %s",
```

Comment:
Don't use String.format, Kotlin has a better method for String interpolation.
```suggestion
        val reply = "${event.member.asMention}: ${openings.random} ${tips.random}"
```